MyTest/src/test/java/com/test/ui/RestAssuredTest.java

```diff
@@ -0,0 +1,49 @@
+package com.test.ui;
+
+import org.hamcrest.Matchers;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+import static pages.ExpectedList.expectedList;
+
+public class RestAssuredTest {
+    private static  final String URL = "http://ergast.com/api/f1/2010/constructors.json";
+    private static final int TOTAL_RECORDS = 12;
+
+    @Test
+    public void isQuantityOfRecordsCorrect(){
+
+        int result = Integer.parseInt(given().when().get(URL).then().extract().path("MRData.total"));
+        Assert.assertTrue(result==TOTAL_RECORDS);
+    }
+
+    @Test
+    public void isConstructorsEqualToExpectList(){
+
+        List<String> result = given().when().get(URL).then().extract().path("MRData.ConstructorTable.Constructors.constructorId");
+        System.out.println(result);
+        Assert.assertTrue(result.equals(expectedList()));
+    }
+
+    @Test
+    public void mercedesValidation(){
+
+        given().when().get(URL).then().assertThat()
```

Comment:
Few things here, first of all you can create Constructor class for equal result, not checking each field separately. And another thing it relates to all tests in class, you can move given().when().get(URL) to before class and store response into variable, because for each test you are triggering endpoint